static/templates/settings/account_settings.hbs

```diff
@@ -8,12 +8,12 @@
                 <form class="email-change-form grid">
                     <div class="input-group">
                         <label class="inline-block title">{{t "Email" }}</label>
-                        <button id='change_email' type="button" class="button btn-link small rounded inline-block" {{#unless user_can_change_email}}disabled="disabled"{{/unless}}>
-                            {{page_params.delivery_email}}
-                            <i class="fa fa-pencil"></i>
-                        </button>
-                        <i class="tippy-zulip-tooltip fa fa-question-circle change_email_tooltip settings-info-icon" {{#if user_can_change_email}}style="display: none;"{{/if}} data-tippy-content="{{t 'Email address changes are disabled in this organization.' }}">
-                        </i>
+                        <div id="change_email_container" {{#unless user_can_change_email}}class="inline-block change_email_tooltip"{{else}}class="inline-block"{{/unless}}>
+                            <button id='change_email' type="button" class="button btn-link small rounded inline-block" {{#unless user_can_change_email}}disabled="disabled"{{/unless}}>
```

Comment:
Would be better to use double quotes for ID here just for consistency.

Reply:
When updating this in a prep commit, I also took the opportunity to make the ids more clear / specific.

So, the button id is now `change_email_button` (instead of `change_email`) and the form id for the modal is `change_email_form` (instead of `change_email_container`). 

And the id for the new `<div>` wrapper for the tooltip in the 3rd commit is `change_email_button_container`.

Reply:
Definitely an improvement, thanks!